lib/src/model/uuid.dart

```diff
@@ -3,6 +3,7 @@ import 'dart:typed_data';
 import 'package:collection/collection.dart';
 import 'package:meta/meta.dart';
 
+/// Representation of a 128-bit number used to identify BLE services and characteristics
```

Comment:
That is not exactly true. The standard ID is indeed a 128-bit lump, but in BLE shorter 16- and 32-bit formats are also used. The the length is checked in `Uuid.parse(·)`, but not in `Uuid(·)`, so it's possible construct an instance with arbitrary payload, actually.